src/app/src/app.reducers.js

```diff
@@ -7,12 +7,14 @@ import {
     deselectSensor,
     showSensorModal,
     hideSensorModal,
+    setSensorData,
 } from './app.actions';
 
 const initialAppState = {
     isIntroVisible: true,
     selectedSensor: null,
     isSensorModalDisplayed: false,
+    sensorData: null,
```

Comment:
At first I thought we should store this data with the `sensors` key on the `map` state, but on second thought this actually makes more sense. Good call!